### ISSUE TITLE
Official source for a recent NAVADMIN isn't listed

### ISSUE THREAD

**Issue by mpyne-navy** [mpyne-navy/navadmin-scanner#4]
The official source link for [NAVADMIN 135/23](https://navadmin-viewer.fly.dev/NAVADMIN/135/23) doesn't show up for some reason.  Other NAVADMINs before and after are fine.

Probably a minor bug (maybe the .txt file got renamed and doesn't match metadata?) but I'm not going to look into it now and don't want to forget it.

**Comment by mpyne-navy** [comment]
```sh
$ ./download-updated-navadmins.pl 
Loaded NAVADMINs for https://www.mynavyhr.navy.mil/References/Messages/NAVADMIN-2023/
Downloading and updating 144 NAVADMIN messages
Downloaded NAVADMIN/NAV23145.txt
Downloaded NAVADMIN/NAV23144.txt
Downloaded NAVADMIN/NAV23143.txt
Downloaded NAVADMIN/NAV23142.txt
Downloaded NAVADMIN/NAV23141.txt
Inactivity timeout at ./download-updated-navadmins.pl line 121.
```

Noticed this today when updating NAVADMINs.  I believe the inactivity timeout causes this issue because I noticed that the `navadmin_meta.json` was left unchanged after this.  Forcing the script to re-run ended up working with no inactivity timeout, and this time `navadmin_meta.json` was properly updated.

**Comment by mpyne-navy** [comment]
Fixed in commit c9fdfc5 (which mistakenly thought this was issue #1)
